packages/connector-aliyun-dm/tsconfig.base.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@silverhand/ts-config/tsconfig.base",
+  "compilerOptions": {
+    "outDir": "lib",
```

Comment:
no need for this line as it's default value

Reply:
Per testing, this is needed otherwise compiled files appeared in `src/`.
<img width="497" alt="image" src="https://user-images.githubusercontent.com/15182327/165268518-cb547af8-1347-44a0-9e63-223d39808ea0.png">